### PR TITLE
Followup for test runner of integration tests

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
Unfortunately I hadn't paid the attention when I was merging #67 and accidentally merged #66 cause 67 was based on it. So, @zazabe please review #66 and those minor changes. Your notes will be handled in this PR.

File: README.md [hunk 1]
```diff
@@ -514,8 +514,14 @@ The library is available for Node and Browser environment. In Browser it is decl
     Creates a new instance of ConnectionError. Extends JanusError.
     * `janusMessage` {Object} message that caused the error.
 
-## Test
-There are unit and integration tests. Both are included into CI. To run them just use `npm test`. Also you can check integration tests locally. Just load `test/integration/index.html` into the latest version of Chrome.
+## Tests
+Tests are located under `test/` directory. To tun them use `npm test`.
```
Comment (zazabe) [inline README.md]: "To run"

Comment (vogdb) [inline README.md]: oops >_<

**Comment by zazabe** [comment]
looks very good, I ❤️  this `karma`!

btw, I tried a `npm test` in the box:
```
  84 passing (6s)
  1 failing

  1) WebsocketConnection tests fails to open invalid address:
     Error: timeout of 2000ms exceeded
      at Timeout.<anonymous> (/home/vagrant/janus-gateway-js/node_modules/mocha/lib/runnable.js:158:19)
      at tryOnTimeout (timers.js:224:11)
      at Timer.listOnTimeout (timers.js:198:5)
```

Is there anything to do for this?

File: README.md [hunk 2]
```diff
@@ -514,8 +514,14 @@ The library is available for Node and Browser environment. In Browser it is decl
     Creates a new instance of ConnectionError. Extends JanusError.
     * `janusMessage` {Object} message that caused the error.
 
-## Test
-There are unit and integration tests. Both are included into CI. To run them just use `npm test`. Also you can check integration tests locally. Just load `test/integration/index.html` into the latest version of Chrome.
+## Tests
+Tests are located under `test/` directory. To run them use `npm test`.
+
+### Unit
+These tests are located under `test/unit/`. To run them use `node_modules/mocha/bin/mocha`.
```
Comment (zazabe) [inline README.md]: why not `$(npm bin)/` instead of `node_modules/mocha/bin`, same for `karma`?

Comment (vogdb) [inline README.md]: fixed

**Comment by vogdb** [comment]
> Is there anything to do for this?

Can't reproduce. Have you tried to rerun? Reinstall npm deps?

**Comment by zazabe** [comment]
I tried with a fresh box, after a `git clean -fd` (no `node_modules` anymore) and I got the same issue.
To fix it, I must run the `$(npm bin)/mocha` command at least once on my laptop (not in the vagrant box), and then it works inside the box again!

I suspect `config.vm.synced_folder` Vagrant config to be the reason... 

I would say let's merge it, because CI is green anyhow!!

**Pull request merged**